src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,85 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuGroupRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class MenuGroupServiceTest {
+
+    @Mock
+    private MenuGroupRepository menuGroupRepository;
+
+    private MenuGroupService menuGroupService;
+
+    @BeforeEach
+    void setUp() {
+        menuGroupService = new MenuGroupService(menuGroupRepository);
+    }
+
+    @DisplayName("`메뉴 그룹`을 생성할 수 있다.")
+    @Test
+    void createMenuGroupWithValidInput() {
+        // given
+        MenuGroup given = new MenuGroup();
+        given.setName("메뉴 그룹 이름");
+
+        // when
+        when(menuGroupRepository.save(any())).then(invocation -> invocation.getArgument(0));
```

Comment:
then을 사용하신 이유가 있을까요? 👀
then 보다는 `thenReturn`을 사용하는 것이 조금 더 명확하지 않을까 싶어요. 🤔

```suggestion
        when(menuGroupRepository.save(any())).thenReturn(expected);
```

Reply:
안녕하세요 @Rok93 ,
메뉴 그룹을 생성할때 request 인스턴스로 리턴하는 것이 아닌, 내부에서 UUID 도 만들어주고 반환하기 때문에 then(invocation -> ); 을 사용했습다!